frontend/src/components/customers/CustomerReservationServices.js

```diff
@@ -11,11 +14,21 @@ function CustomerReservationServices(props) {
           <Row>
             {props.services.map((service, i) => {
```

Comment:
Since this seems to almost entirely depend on a prop, it may be useful to check if props.services isn't null or empty or something before doing the map.

Reply:
Done. Must add proper error handling later on but for now just used a redirect, though this prop being empty would mean there's a problem with the backend.